ENGAGEHF/Managers/MessageManager/MessageManager.swift

```diff
@@ -63,17 +67,74 @@ final class MessageManager: Manager {
         if let account, account.signedIn {
             updateSnapshotListener(for: account.details)
         }
+        
+        startProcessingCleanupTimer()
+    }
+    
+    private func startProcessingCleanupTimer() {
```

Comment:
I am not sure why this function exists in addition to the timers that are fired in the function that registers the processing state (and should probably be moved to the processing state type). As far as I can see this seems redundant.